README.md

```diff
@@ -175,75 +186,44 @@ to change s
-        ~<list>.append(<elem>)
-        ~<list>.insert(<index>,<elem>)
-    Remover un elemento
-        <list>.remove(<elem>)
-    Find an element in the list
-        <elem> in <list>
-    .index(<elem>)para el indice
-        <list>.index(<elem>)
-    Change an  element
-        <list>[<índex>] = <new_value>
-    Methodology of the list
-        mutual operations of the lists
-        <list>.<method>(<parameters>)
-        .count(<elem>)
-    
-    ​    .extend(<list>)
-    
-    ​    .pop()
-    
-    ​    .reverse()
-    
-    ​    .sort()
+**Definition:** Arrays are a **MUTABLE** kind of variable, which can store multiple objects inside it, organized by indexes.
 
-### String
+> Mutable means it can be later modified.
 
-Strings are basically lists, and as so, they have all the previously seen properties (indexation and slicing).
+- An array can contain objects, which can be of every kind.
+- An array can also contain objects created before.
+- It stores elements in sequence.
+
+In the example below, we have an array called `my_array` that was created with three elements created before and 2 elements (a string and a boolean) created inside it:
+
+```
+element_1 = 4.4 # float
+element_2 = 5 # int
+element_3 = [6, 4, 7] # array
+
+my_array = [element_1, element_2, element_3, 'Hello', False]
+```
+
+- **Indexation:** An array index it's elements starting from number `0`. It is specially useful to know because some problems began when you think the first element is the element number 1.
+- **Acces:** To acces any item in an array, you need it's index. For example, you can write `my_array[0]` to acces the first element
+- **Slice:** To slice an array, you can use the following notation: `[ starting_slice : ending_slice : jump ]`. So, for example if you want to slice an array from the start to the element in index number 5, and you want every 2 elements, you can write `my_array[:5:2]`.
+- **Length:** To get the length of an array, use the `len()` function and pass the array as a parameter. Example: `len(my_array)`.
+
+Aside from those uses of arrays, they have some methods shich can be found on the [Official Python Docs](https://docs.python.org/3/library/array.html). The most important ones are:
+
+- **Append:** Add an element at the end of the array.
+- **Insert:** Inserts an element at any index of the array.
+- **Pop:** Removes the last element and then returns it.
+
+### String
 
-In addition to those, `Strings` have special methods than can be used following the next example: <cadena>.<method>(<values>)
+**Definition:** Strings are basically **UNMUTABLE** lists, and as so, they have some of the previously seen properties, like indexation and slicing. Since strings are unmutable, when you want to make a change to a string, you must create a new and different one.
 
- Some of the special String methods are:
+In addition to those, `Strings` have special methods than can be used following the next example: <string>.<method>(<values>)
```

Comment:
special methods 'than' can
than or is it that???

Reply:
fixed